app/code/Magento/PageBuilder/view/adminhtml/web/ts/js/binding/live-edit.ts

```diff
@@ -33,8 +33,8 @@ function stripHtml(html: string) {
  * @param {Element} element
  */
 function handlePlaceholderClass(element: Element) {
-    if (stripHtml(element.innerHTML).length === 0) {
-        element.innerHTML = "";
+    if (stripHtml(element.textContent).length === 0) {
```

Comment:
Please remove the usage of `stripHtml` here because we are using `textContent` here which works same.

Reply:
Done